3scale/porta
# Better Flow support for React files

**What this PR does / why we need it**:

Implements Flow support in Dashboard (ApiFilter) and Navigation (ContextSelector) modules and set correctly some options in `.flowconfig` file.

**Verification steps** 

All tests should pass and no Flow errors should pop up:
1. Install dependencies and typings:
`$ npm i # Use npx or npm-run`
2. Run tests and checks
`$ npm test && npm run jest`
